basics.js

```diff
@@ -31,6 +40,18 @@ def greet(name, excited)
 end
 */
 
+var greet = function(name, excited){
+  var greeting = ("Hi, " + name + " here!");
+  if (excited === true) {
```

Comment:
No need to check that `excited === true` as it will automatically return a true/false value. But still correct.